Source/ResourceLib/Accelerator.cs

```diff
@@ -45,7 +45,58 @@ public string Key
             get
             {
                 string key = Enum.GetName(typeof(User32.VirtualKeys), _accel.key);
-                return string.IsNullOrEmpty(key) ? ((char) _accel.key).ToString() : key;
+                return string.IsNullOrEmpty(key) ? ((char)_accel.key).ToString() : key;
+            }
+            set
+            {
+                try//will work if given string is a Virtual Key e.g. "VK_NUMPAD1"
+                {
+                    uint i = (uint)Enum.Parse(typeof(User32.VirtualKeys), value);
+                    _accel.key = (UInt16)i;
+                }
+                catch (Exception e)//otherwise, use ASCII-code of given Key e.g. key "Z" == 90
+                {
+                    char c = value.ToCharArray()[0];
+                    UInt16 i = (UInt16)c;
+                    _accel.key = i;
+                }
+            }
+        }
+
+        /// <summary>
+        /// String representation of the used flags.
+        /// When using set method, several flags have to be split with ',' e.g. "VIRTKEY, NOINVERT, CONTROL"
+        /// get method will return flags in the same way, e.g. "VIRTKEY,SHIFT,ALT"
+        /// </summary>
+        public string Flags

```

Comment:
I don't think this should be a string method that parses comma-separated values. It should be an unsigned integer property that you can assign `AcceleratorVirtualKey` values to. Or maybe it should be a collection of these. Either way, String seems unnecessarily complicated and very error prone here.